lib/src/widgets/coin_search.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+
+import '../config/app_constants.dart';
+import '../constants/constants.dart' as constant;
+import '../models/crypto.dart';
+
+class CoinSearch extends StatelessWidget {
+  const CoinSearch({Key? key, this.dataCrypto}) : super(key: key);
+
+  final List<Crypto>? dataCrypto;
+
+  static String _displayStringForOption(Crypto option) => option.name!;
+
+  @override
+  Widget build(BuildContext context) {
+    double optionsViewBuilderwidth = MediaQuery.of(context).size.width * 0.9;
+    double containerMargin = MediaQuery.of(context).size.height * .0275;
+   
+    return Container(
+      margin: EdgeInsets.symmetric(horizontal: containerMargin),
+      decoration: const BoxDecoration(
+        color: Colors.white,
+      ),
+      child: Autocomplete<Crypto>(
+        displayStringForOption: _displayStringForOption,
+        fieldViewBuilder: (BuildContext context,
+            TextEditingController fieldTextEditingController,
+            FocusNode fieldFocusNode,
+            VoidCallback onFieldSubmitted) {
+          return TextField(
+            decoration: const InputDecoration(
+                prefixIcon: Icon(Icons.search),
+                border: OutlineInputBorder(),
+                hintText: constant.SearchScreen.hintText),
+            controller: fieldTextEditingController,
+            focusNode: fieldFocusNode,
+            style: const TextStyle(color: Colors.black, fontSize: 18),
```

Comment:
why didn't you use Theme?

Reply:
i fixed it, thank you. I'm already know it now